src/commitlint/index.spec.ts

```diff
@@ -44,24 +69,84 @@ describe('@co-it/schematics:commitlint', () => {
     });
   });
 
-  describe('When package.json has no husky entry', () => {
+  describe('When .huskyrc does not exist', () => {
     beforeEach(() => {
       const packageBeforeInstall = { devDependencies: {} };
       treeBefore.create('package.json', JSON.stringify(packageBeforeInstall));
     });
 
-    it('should add an husky entry with commitlint', () => {
+    it('should configure a commit-msg hook in .huskyrc', () => {
       const treeAfter = runner.runSchematic('commitlint', {}, treeBefore);
```

Comment:
I like the naming convention you came up with.
We should apply this to every test.

I started to use: `project` ➡️ `treeBefore`, `tree` ➡️ `treeAfter`
I believe your naming is more convenient. 👍